docs/extras/updating.md

```diff
@@ -4,12 +4,17 @@ This page documents how you can keep your system up-to-date.
 
 After following our guide, your system will consist of three core elements that can be updated. Atmosphere, Hekate and your system firmware.
 
-### **Updating Atmosphere**
+## Updating Atmosphere
 
 When updating Atmosphère, always make sure to _read the release notes_. They may list important changes and modifications to your system.
 
-!!! warning "Updating from below Atmosphère 1.0.0"
-    If you update from below Atmosphère 1.0.0, there are additional steps to follow. You will have to delete the `sept` folder from your microSD, delete `fusee-secondary.bin` from your `atmosphere` folder and update your Hekate config file: <a href="../../files/emu/hekate_ipl.ini" download>hekate_ipl.ini</a> in the `bootloader` folder.
+::: warning
+
+**Updating from below Atmosphère 1.0.0**
```

Comment:
Unrelated to this PR, but we're on atmosphere 1.8.0. I don't expect anyone to still be on something below 1.0.0.